proposals/lip-0045.md

```diff
@@ -376,7 +380,7 @@ channelSchema = {
                     "fieldNumber": 1
                 },
                 "size": {
-                    "dataType": "uint64",
```

Comment:
Why don't we use `uint64` everywhere here? In terms of storage it will be the same as long as we are in the `uint32` range. Note that we could probably have 1 M CCMs per day => 3600 M CCMs in 10 years.

Reply:
This comes from a discussion we had with the SDK to ensure that it fits into `number` type and we can avoid using `bigint` for the moment. If this becomes a problem in the future, we can upgrade to uint64 and bigints. Maybe @shuse2 has some extra input?

Reply:
Current tree takes uint32, and if we change to uint64, we need to update to use `bigint` where we have difference in the performance and extra workload.
That's why we suggested to go with uint32 for now, and update later if needed

Reply:
It is true that it is somewhat inconsistent with what we have in LIP 0031: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0031.md#proof-serialization

Reply:
Okay, if this is better for the implementation, sounds good to me. Not sure if we should update LIP 0031. The comment can be resolved from my side.

Reply:
I would leave it as is for the moment then.

Reply:
So we can resolve the comment?

Reply:
yes, resolving.